tests/test_quickstart.py

```diff
@@ -20,7 +20,7 @@ def mock_term_input(prompt):
             try:
                 return next(generator)
```

Comment:
I've kept it as it is, to not loose the focus of this PR, for now just moving over the flake8; next PR would be to fix code styles 👍  trying to keep changes small and easy to review